WordPress/src/main/java/org/wordpress/android/util/extensions/ViewExtensions.kt

```diff
@@ -83,3 +83,15 @@ fun RecyclerView.disableAnimation() {
 }
 
 fun View.getString(@StringRes stringRes: Int) = context.getString(stringRes)
+
+fun View.visible() {
+    this.visibility = View.VISIBLE
+}
+
+fun View.gone() {
+    this.visibility = View.GONE
+}
+
+fun View.invisible() {
+    this.visibility = View.INVISIBLE
+}
```

Comment:
💡 `androidx` already provides similar extensions that are used throughout the app code already (`isVisible`, `isGone`, `isInvisible`), so I think we should keep using them instead of adding new extensions.